ansible/roles/operator-pipeline/tasks/pipeline-secrets.yml

```diff
@@ -211,6 +211,31 @@
         username: "{{ iib_quay_username | b64encode }}"
         password: "{{ iib_quay_password | b64encode }}"
 
+- name: |
+    Create community operator quay credentials secret for pushing final index into;
+    PROD - quay.io/repository/redhat-pending/redhat----community-operator-index
```

Comment:
Can you add these references as a comment to the task? The name is visible in the ansible-playbook run an multiline names will mess up the output.